build.gradle

```diff
@@ -64,6 +66,8 @@ subprojects {
 
     dependencies {
         implementation "org.projectlombok:lombok:${lombokVersion}"
+        implementation "co.elastic.clients:elasticsearch-java:${elasticsearchVersion}"
```

Comment:
Do we want to include this within Marquez jar or let users add the jar manually in case they turn on the feature? The benefit of the latter approach is that it's easier to switch to another implementation if needed.